src/main/java/loanbook/logic/commands/SetEmailCommand.java

```diff
@@ -0,0 +1,80 @@
+package loanbook.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static loanbook.commons.util.CollectionUtil.requireAllNonNull;
+
+import loanbook.commons.core.Messages;
+import loanbook.logic.CommandHistory;
+import loanbook.logic.commands.exceptions.CommandException;
+import loanbook.model.Model;
+
+/**
+ * Set user's email to the app.
+ */
+public class SetEmailCommand extends Command {
+
+    public static final String COMMAND_WORD = "setemail";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Set your email to the app. "
+            + "Please note that the default email is <default>!\n"
+            + "Parameter: OLDEMAIL NEWEMAIL\n"
+            + "Example: " + COMMAND_WORD + " default myemail@gmail.com";
+
+    public static final String MESSAGE_SUCCESS = "Your email is set successfully!";
+
+    private final String oldEmail;
+    private final String newEmail;
+
+    /**
+     * Creates an SetEmailCommand to set user's {@code Email} according to the {@code newEmail} provided.
+     */
+    public SetEmailCommand(String oldEmail, String newEmail) {
+        requireAllNonNull(oldEmail, newEmail);
+        this.oldEmail = oldEmail;
+        this.newEmail = newEmail;
+    }
+
+    /**
+     * Check if user's new {@code Email} is a valid Gmail.
+     */
+    public boolean isValidGmail(String email) {
```

Comment:
since String email is an email, we should cast it to a `Email` object and do `#isValidGamil` from the Email class.

Reply:
i.e. We should shift this to the `Email` class.